src/CommunityToolkit.Maui.SourceGenerators/Diagnostics/TextColorToDiagnostic.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace CommunityToolkit.Maui.SourceGenerators
+{
+	static class TextColorToDiagnostic
+	{
+		const string category = "TextColorTo";
+		public static readonly DiagnosticDescriptor GlobalNamespace = new(
+			   "MCT001",
+			   "Global namespace is not support for this Source Generator",
```

Comment:
Users may have no context about what "this" refers to.

```suggestion
			   "Global namespace is not supported for 'TextColorTo' Source Generator",
```